File: app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/model/NoteRemote.java
package br.org.cesar.discordtime.stickysessions.data.remote.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class NoteRemote {

    @SerializedName("id")
    @Expose
    public String id;
    @SerializedName("description")
    @Expose
    public String description;
    @SerializedName("user")
    @Expose(serialize = false)
    public String user;
    @SerializedName("topic")
    @Expose
    public String topic;
    @SerializedName("session_id")
    @Expose
    public String sessionId;

    public NoteRemote(){}
}

File: app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/model/NoteRemote.kt
package br.org.cesar.discordtime.stickysessions.data.remote.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class NoteRemote(
        @Expose @SerializedName("id")
        var id: String? = "",
        @Expose @SerializedName("description")
        var description: String? = "",
        @Expose(serialize = false) @SerializedName("user")
        var user: String? = "",
        @Expose @SerializedName("topic")
        var topic: String? = "",
        @Expose @SerializedName("session_id")
        var sessionId: String? = ""
)